website/docs/tutorials/07-testing/02-cheatcodes/expect-events.md

```diff
@@ -57,3 +53,64 @@ func test_expect_event_by_contract_address{syscall_ptr: felt*, range_check_ptr}(
     return ();
 }
 ```
+
+:::tip
+You can provide `"data"` as a dictionary to leverage [data transformer](README.md#data-transformer) - see example below
+:::
+
+
+```cairo title="Emitting a complexly-strucutred event, and expecting it in tests using data-transformer"
```

Comment:
```suggestion
```cairo title="Emitting a complex structured event, and expecting it in tests using data transformer"
```